packages/eas-cli/src/submit/ArchiveSource.ts

```diff
@@ -304,38 +305,31 @@ async function handleBuildListSourceAsync(
 function formatBuildChoice(build: BuildFragment, expiryDate: Date): prompts.Choice {
   const {
     id,
-    platform,
     updatedAt,
-    appVersion,
-    sdkVersion,
     runtimeVersion,
     buildProfile,
-    appBuildVersion,
     releaseChannel,
-    initiatingActor,
+    gitCommitHash,
+    gitCommitMessage,
   } = build;
 
   const formatValue = (field?: string | null): string =>
     field ? chalk.bold(field) : chalk.dim('Unknown');
 
   const buildDate = new Date(updatedAt);
-  const maybeRuntimeVersion = runtimeVersion ? `Runtime: ${formatValue(runtimeVersion)}` : null;
-  const maybeSdkVersion = sdkVersion ? `SDK: ${formatValue(sdkVersion)}` : null;
-  const appBuildVersionString = `${
-    platform === AppPlatform.Android ? 'Version code' : 'Build number'
-  }: ${formatValue(appBuildVersion)}`;
+
+  const formatedCommitData =
```

Comment:
```suggestion
  const formattedCommitData =
```

Reply:
Just checked the prompts library [documentation](https://www.npmjs.com/package/prompts#selectmessage-choices-initial-hint-warn) and it seems like we can't get rid of the underline effect using its API 🤔. I assumed it is possible, although in reality, it's not.